src/test/rgw/CMakeLists.txt

```diff
@@ -140,7 +140,7 @@ add_ceph_unittest(unittest_rgw_string)
 # unitttest_rgw_dmclock_queue
 add_executable(unittest_rgw_dmclock_scheduler test_rgw_dmclock_scheduler.cc $<TARGET_OBJECTS:unit-main>)
 add_ceph_unittest(unittest_rgw_dmclock_scheduler)
-target_link_libraries(unittest_rgw_dmclock_scheduler ${rgw_libs} dmclock ${Boost_LIBRARIES})
+target_link_libraries(unittest_rgw_dmclock_scheduler radosgw_a dmclock ${Boost_LIBRARIES})
```

Comment:
note the changes from https://github.com/ceph/ceph/pull/26799, which removed `${Boost_LIBRARIES}`. does it link without the `dmclock` bit too?

Reply:
checking. I just added another commit as librgw includes were not linking dmclock, ceph_test_librgw_file_marker was failing.